facet-jsonschema/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "facet-jsonschema"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+repository.workspace = true
+
+[dependencies]
+num-traits = { version = "0.2.19", default-features = false }
+facet-core = { version = "0.5.3", path = "../facet-core" }
+facet-reflect = { version = "0.6.2", path = "../facet-reflect" }
+facet = { path = "../facet" }
+
+[dev-dependencies]
+#serde_json = "1.0"
+#schemars = { version = "0.8" }
```

Comment:
You know, I think eventually it will relax the constraint and it's: no syn as a regular dependency, but as a dev dependency it's okay.

We're going to want to compare the facet stuff against the existing ecosystem

Reply:
Yeah, I figured as much. For this particular case, it was useful to test the parsing and compare the output and derive from `schemars`.